CHANGELOG.md

```diff
@@ -7,122 +7,74 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- Fix redundant entry in lockfile ([#14616](https://github.com/MetaMask/metamask-extension/pull/14616))
-- Jest tests - incremental coverage ([#14612](https://github.com/MetaMask/metamask-extension/pull/14612))
-- build - update bify-module-groups for build determinism ([#14610](https://github.com/MetaMask/metamask-extension/pull/14610))
+- Add Coinbase Pay as fiat onramp option ([#14648](https://github.com/MetaMask/metamask-extension/pull/14648))
+- Add networks without ticker trait, for better analytics ([#14607](https://github.com/MetaMask/metamask-extension/pull/14607))
+- Add event tracking for Token Detection V2 ([#14441](https://github.com/MetaMask/metamask-extension/pull/14441))
 - Add search information for Theme dropdown ([#14476](https://github.com/MetaMask/metamask-extension/pull/14476))
-- Update version parsing to allow rollback release ([#14288](https://github.com/MetaMask/metamask-extension/pull/14288))
-- Update bug-report.yml ([#14466](https://github.com/MetaMask/metamask-extension/pull/14466))
-- Disable STX if a regular tx is in progress ([#14554](https://github.com/MetaMask/metamask-extension/pull/14554))
-- Fix development build scripts ([#14594](https://github.com/MetaMask/metamask-extension/pull/14594))
-- Update PULL_REQUEST_TEMPLATE.md ([#14547](https://github.com/MetaMask/metamask-extension/pull/14547))
-- Patch gridplus-sdk for perf fix ([#14582](https://github.com/MetaMask/metamask-extension/pull/14582))
-- Add applyLavaMoat build flag ([#14583](https://github.com/MetaMask/metamask-extension/pull/14583))
-- fix cross-fetch moderate vulnerability alert ([#14570](https://github.com/MetaMask/metamask-extension/pull/14570))
-- Fix browser.tab.get call ([#14550](https://github.com/MetaMask/metamask-extension/pull/14550))
-- Added getAccounts suppressUnauthorized param ([#14126](https://github.com/MetaMask/metamask-extension/pull/14126))
+
+### Changed
+- Update caret icon color on Home Page for darkmode, from black to white ([#14835](https://github.com/MetaMask/metamask-extension/pull/14835))
+- Code cleanup around tx logic: constants/transactions, `useTransactionDisplayData`, and `TransactionIcon` ([#14784](https://github.com/MetaMask/metamask-extension/pull/14784))
+- Removed draft transaction object from the send slide, for avoiding sync errors ([#14701](https://github.com/MetaMask/metamask-extension/pull/14701))
+- Update "secp256k1" for fast module init via lazy loading ([#14677](https://github.com/MetaMask/metamask-extension/pull/14677))
+- Update the color of the ledger alert text to adhere to design system guidelines ([#14674](https://github.com/MetaMask/metamask-extension/pull/14674))
+- Update version parsing to allow rollback release on Chrome ([#14288](https://github.com/MetaMask/metamask-extension/pull/14288))
+- Make existing component `SiteOrigin` reusable across all contexts where we surface a connected site's origin ([#14720](https://github.com/MetaMask/metamask-extension/pull/14720))
```

Comment:
Suggestion to change "Make existing component `SiteOrigin` reusable across all contexts where we surface a connected site's origin" to "Standardize display of connected site's origin data on all confirmation screens."

This is because the end user doesn't know what a "component" is, so we should leave that concept out of the changelog

Reply:
aha this makes sense. I will consider this for future RCs, thank you